doggo_description:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  
  
  # Position Controllers ---------------------------------------
  leg_controller1:
    type: effort_controllers/JointPositionController
    joint: frame_leg11
    pid: {p: 6, d: 0.1, i: 0.1}

  leg_controller2:
    type: effort_controllers/JointPositionController
    joint: frame_leg12
    pid: {p: 6, d: 0.1, i: 0.1}

  leg_controller3:
    type: effort_controllers/JointPositionController
    joint: frame_leg21
    pid: {p: 6, d: 0.1, i: 0.1}

  leg_controller4:
    type: effort_controllers/JointPositionController
    joint: frame_leg22
    pid: {p: 6, d: 0.1, i: 0.1}

  leg_controller5:
    type: effort_controllers/JointPositionController
    joint: frame_leg31
    pid: {p: 6, d: 0.1, i: 0.1}

  leg_controller6:
    type: effort_controllers/JointPositionController
    joint: frame_leg32
    pid: {p: 6, d: 0.1, i: 0.1}

  leg_controller7:
    type: effort_controllers/JointPositionController
    joint: frame_leg41
    pid: {p: 6, d: 0.1, i: 0.1}

  leg_controller8:
    type: effort_controllers/JointPositionController
    joint: frame_leg42
    pid: {p: 6, d: 0.1, i: 0.1}